src/popup/components/OverviewDetailsParty.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div
+    class="overview-details-party"
+    :data-cy="isRecipient ? 'recipient' : 'sender'"
```

Comment:
In most cases element IDs used for testing cases should not be dynamic. I suggest to apply the data-cy properties on parent level to avoid being dynamic.